### PR TITLE
HoC 2023 - Code.org - Update code.org/hourofcode

### PR DESCRIPTION
**Pull request by dmantonyuk**
_(This PR description is authored by @kelbyhawn)_

Update https://code.org/hourofcode
- Page is fully responsive
- Updated OpenGraph tags and image
- Redirected and deprecated https://code.org/hourofcode/overview to https://code.org/hourofcode
- Updated outdated `/hourofcode/overview` links

**Jira ticket:** https://codedotorg.atlassian.net/browse/ACQ-995

----

localhost code org_3000_hourofcode

### CODE REVIEW

**Comment by dmantonyuk** [comment]
New HOC

File: pegasus/sites.v3/hourofcode.com/i18n/en.yml [hunk 1]
```diff
@@ -502,6 +516,7 @@
   module_grade_2_5: "2-5"
   module_grade_2_8: "2-8"
   module_grade_2_12: "2-12"
+  module_grade_2_13: "2-13"
```
Comment (hannahbergam) [inline pegasus/sites.v3/hourofcode.com/i18n/en.yml]: Is this for a grade range of 2-13? And will we use this somewhere else?

File: pegasus/sites.v3/code.org/public/hourofcode/overview.haml [hunk 2]
```diff
@@ -1,80 +0,0 @@
----
-title: Hour of Code Teacher Resources
-theme: responsive
----
-
-:css
-  .container_responsive { width: 970px }
-
-%link{href: "/css/student.css", rel: "stylesheet"}
```
Comment (hannahbergam) [inline pegasus/sites.v3/code.org/public/hourofcode/overview.haml:9]: Is any of this being moved around or is all the new code new?

**Review by hannahbergam — COMMENTED** [review]
The copy looks great! Just a couple small questions

File: pegasus/sites.v3/hourofcode.com/i18n/en.yml (hunk 1 above)
Comment (kelbyhawn) [reply pegasus/sites.v3/hourofcode.com/i18n/en.yml]: Oh great catch! This is a typo, I'll remove it.

File: pegasus/sites.v3/code.org/public/hourofcode/overview.haml (hunk 2 above)
Comment (kelbyhawn) [reply pegasus/sites.v3/code.org/public/hourofcode/overview.haml:9]: All the new code is new. We're essentially deprecating this `overview.haml` page and using [this new](url) code.org/hourofcode (`index.haml`) as the new main hoc page on code.org.

**Review by hannahbergam — APPROVED** [review]
LGTM! Thanks for the quick answers to my questions!

**Pull request merged**